node/src/main/scala/coop/rchain/node/api/grpc.scala

```diff
@@ -35,23 +40,28 @@ object GrpcServer {
     Task.delay {
       NettyServerBuilder
         .forPort(port)
-        .executor(scheduler)
+        .executor(grpcExecutor)
```

Comment:
What is this `executor` used for? You set manually the `grpcExecutor` betlow for `ReplGrpcMonix` and `DiagnosticsGprcMonix` manually so for which calls is, this `executor` call, setting the scheduler?

Reply:
This is `grpc-java` specific. The executor is used internally for incoming requests. If we don't set here an `executor` then `grpc-java` will create its own `ForkJoinPool`. We want to use the same thread pool for all gRPC IO in `ReplGrpcMonix`, `DiagnosticsGprcMonix`, `DeployGrpcService` and `TcpTransportLayer`.